spec/requests/flows/authorization_code_spec.rb

```diff
@@ -223,13 +223,7 @@ def authorize(redirect_url)
         visit authorization_endpoint_url(client: @client)
         click_on "Authorize"
 
-        authorization_code = current_params["code"]
-        create_access_token authorization_code, @client, code_verifier
-
-        expect(json_response).to match(
-          "error" => "invalid_grant",
-          "error_description" => translated_error_message(:invalid_grant),
-        )
+        url_should_have_param("code", Doorkeeper::AccessGrant.first.token)
```

Comment:
I'm a bit confused by the PKCE specs in this file because it seems we don't mock `Doorkeeper::AccessGrant).pkce_supported?`, or set it up through other ways.

Should we add this?

Reply:
I think yes

Reply:
@nbulaj I saw now that we indirectly set this up by adding the relevant columns: https://github.com/toupeira/doorkeeper/blob/4a7a106dc03411d9ada0a627f6b46e9e8022c393/spec/dummy/db/schema.rb#L25-L28

Running these specs with `WITHOUT_PKCE=1` makes them fail, so I think we're good here and I retract my concern :grin: